### PR TITLE
Extract standard names from variables in netcdf file and add as CFSTDN keywords

### LINKED ISSUE #240: Automatically add CFSTDN keywords from the variables
The variables inside netcdf-cf files contain cf standard names. `nc_to_mmd` should pull out the `standard_names` and add them as keywords.

### PR DESCRIPTION
**Pull request by shamlymajeed**
- Extracted standard names from variables.

 - Added as CFSTDN keywords

**Summary**: closes #240 

**Related issue**: 

**Suggested reviewer(s)**:

**Reviewer checklist**:

- [ ] The headers of all files contain a reference to the repository license (i.e., "License: This file is part of py-mmd-tools, licensed under the Apache License 2.0 (https://www.apache.org/licenses/LICENSE-2.0)")
- [ ] 100% test coverage of new code - meaning:
    - [ ] The overall test coverage increased or remained the same as before
    - [ ] Every function is accompanied with a test suite
    - [ ] Tests are both positive (testing that the function work as intended with valid data) and negative (testing that the function behaves as expected with invalid data, e.g., that correct exceptions are thrown)
    - [ ] Functions with optional arguments have separate tests for all options
- [ ] Examples are supported by doctests
- [ ] All tests are passing
- [ ] All names (e.g., files, classes, functions, variables) are explicit
- [ ] Documentation (as docstrings) is complete and understandable

The checklist is based on the S-ENDA conventions and definition of done (see https://s-enda-documentation.readthedocs.io/en/latest/general_conventions.html). The above points are not necessarily relevant to all contributions. In that case, please add a short explanation to help the reviewer.

### CODE REVIEW

File: py_mmd_tools/nc_to_mmd.py [hunk 1]
```diff
@@ -681,19 +681,35 @@ def get_personnel(self, mmd_element, ncin):
...
             vocabularies = self.separate_repeated(True, getattr(ncin, acdd_vocabulary_key))
         else:
             ok_formatting = False
             self.missing_attributes['errors'].append(
                 '%s is a required ACDD attribute' % acdd_vocabulary_key
             )
+
+        # add vocabulary CFSTDN
+        if len(cfstd_names) != 0:
+            vocabularies.append('CFSTDN:CF Standard Names:https://vocab.met.no/CFSTDN')
```
Comment (mortenwh) [inline py_mmd_tools/nc_to_mmd.py:711]: the url is wrong. Please commit this suggestion instead:
```suggestion
            vocabularies.append('CFSTDN:CF Standard Names:https://vocab.met.no/mmd/Keywords_Vocabulary/CFSTDN')
```

**Review by mortenwh — CHANGES_REQUESTED** [review]
The url to the CFSTDN term is wrong. Please also make a test to verify that a netcdf file with an attribute containing a CF standard name is registered correctly, i.e., that the keyword is picked up. Also, please use met-vocab-tools to verify that the standard name is actually a cf standard name, and make sure that an error is added to `missing_attibutes` if it is not a valid standard_name.

**Comment by mortenwh** [comment]
@Teddy-1000 - vi får en rar feil her: "'nc_sub' object has no attribute 'standard_name'" - ser ut til at det har å gjøre med endringer som er gjort for å få til api'et. Kan du se på det?

**Comment by Teddy-1000** [comment]
@mortenwh Enesten feilen jeg ser er at det mangler en pakke ipdb. Ser ut som alle 40 testene som feiler har denne feilen. Det er nok ikke reletart til endrigene som ble merget i går. Hva gjøre dere når dere får denne filen? Er det når testene kjøres lokalt?

**Comment by Teddy-1000** [comment]
@mortenwh @shamlymajeed Fikset problemet, alle tester passerer nå.

**Review by mortenwh — APPROVED** [review]

**Pull request merged**